lib/hacluster.pm

```diff
@@ -1134,7 +1134,7 @@ sub calculate_sbd_start_delay {
 =cut
 
 sub setup_sbd_delay() {
-    my $delay = get_var('HA_SBD_START_DELAY', '');
+    my $delay = get_var('HA_SBD_START_DELAY', '60');
```

Comment:
I would rather put it only to job group setting for failing release. If you specify default value here it will set 60s for all releases. 
('HA_SBD_START_DELAY', '' ) means that calculation formula or default value from HA stack is used. For example on 15SP5 or 15SP4 calculated value is 215s and in this case it would be reduced to 60s. Most probably causing test to fail.

Reply:
Done. Set it in job groups 12sp5/15sp1/15sp2/15sp3: https://gitlab.suse.de/qa-css/openqa_ha_sap/-/merge_requests/597